ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml

```diff
@@ -324,6 +324,41 @@ tests:
     - chain: create-infra-move-ingress-monitoring-registry
     workflow: openshift-qe-node-density-cni
   timeout: 5h0m0s
+- always_run: false
```

Comment:
maybe I'm seeing this incorrectly, but aren't you removing this config in the next commit? If so, we can just remove this commit entirely and only use the next one with the real changes coming in